generateEmojiRange.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+
+const UNICODE_EMOJI_LIST_URL =
+  'https://www.unicode.org/Public/UCD/latest/ucd/emoji/emoji-data.txt';
+
+fetch(UNICODE_EMOJI_LIST_URL)
```

Comment:
how do you use this when it's not even exposed as a module. Is this some sort of one-off script?

Reply:
yes, it's a helper for retrieving emoji code range. no needed to be run every time but once when latest emoji list is updated.